src/bun.js/api/bun/subprocess.zig

```diff
@@ -2075,21 +2078,14 @@ pub const Subprocess = struct {
         }
 
         const loop = jsc_vm.eventLoop();
+        var island = bun.new(Island, .{});
```

Comment:
Why is this using `bun.new` instead of `Island.new`?

Why is this called `Island` instead of something like `WeakPtr`? What is an island?

Reply:
doesnt that come from `NewRefCounted` ?

`WeakPtr` seemed like it would infer pointing to only one of them or generic and i didnt want to be misleading

Reply:
> doesnt that come from NewRefCounted ?

You can use `usingnamespace bun.New` adds it as well.